crates/sui-core/src/unit_tests/authority_aggregator_tests.rs

```diff
@@ -1,33 +1,72 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
-use std::collections::BTreeMap;
-use std::path::PathBuf;
-use std::sync::{Arc, Mutex};
-
 use move_core_types::{account_address::AccountAddress, ident_str};
 use move_package::BuildConfig;
 use signature::Signer;
-
+use std::collections::BTreeMap;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+use sui_config::gateway::GatewayConfig;
 use sui_config::genesis::Genesis;
 use sui_config::ValidatorInfo;
 use sui_types::crypto::{
     generate_proof_of_possession, get_key_pair, AccountKeyPair, AuthorityKeyPair,
     AuthorityPublicKeyBytes, SuiKeyPair,
 };
 use sui_types::crypto::{KeypairTraits, Signature};
+use test_utils::authority::SuiNode;
 
 use sui_types::messages::*;
 use sui_types::object::{Object, GAS_VALUE_FOR_TESTING};
+use test_utils::authority::{
+    spawn_test_authorities, test_and_configure_authority_configs, test_authority_configs,
+};
+use test_utils::messages::make_transfer_object_transaction;
+use test_utils::objects::{generate_gas_objects_with_owner, test_gas_objects};
+use test_utils::test_account_keys;
 
 use super::*;
 use crate::authority::AuthorityState;
 use crate::authority_client::{
     AuthorityAPI, BatchInfoResponseItemStream, LocalAuthorityClient,
-    LocalAuthorityClientFaultConfig,
+    LocalAuthorityClientFaultConfig, NetworkAuthorityClient, NetworkAuthorityClientMetrics,
 };
+use crate::gateway_state::GatewayState;
 
 use tokio::time::Instant;
 
+pub async fn init_network_authorities(
```

Comment:
how's this different from `init_local_authorities` below?

Reply:
This one will spin up authorities which will exchange messages over network. The local one is updated inline. I wanted to simulate how it'll be in prod and hence used network authorities.